hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -172,16 +171,13 @@ public void run() {
         }, 10);
     }
 
-    // ignored due to: https://github.com/hazelcast/hazelcast/issues/5035
-    @Ignore
     @Test
-    public void testMapLoaderLoadUpdatingIndex() throws Exception {
+    public void testMapLoaderLoadUpdatingIndex_noPreload() throws Exception {
```

Comment:
Actually, it wasn't. It was testing two things in one test - one with loader, once when you populate the data by yourself. 
I have another branch when the test will maybe with more destroy() oriented way.